docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+---
+title: TLS for Kubernetes
+order: 0
+---
+
+# TLS for Kubernetes
+
+Talking over the internet with no encryption is something unthinkable these days. With Redpanda you can help secure your data by enabling [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) across your clusters, with just a few configurations. 
+
+If you haven’t, please check our guide about [Encryption, authorization & authentication](https://vectorized.io/docs/acls/).
+
+Here we will dive deeper into how we can configure TLS inside a Kubernetes cluster.
+
+The Custom Resource Definition (CRD) of a Redpanda cluster includes three APIs: Kafka, Admin and Pandaproxy APIs. Each API can support TLS for server authentication and mTLS (Mutual TLS) for client authentication.
+
+TLS allows us to establish a secure connection between a client and a broker in which communication is encrypted. 
+
+Mutual TLS (mTLS), or 2-way TLS, is an authentication method in which a client attempting to establish a connection with the server (broker) must present its certificate. This way the server can decide whether the client can be trusted.
+
+## Getting started
+
+To demonstrate how to get up and running with TLS in Kubernetes, we’ll use a sample `yaml` configuration file for a one-node cluster.
```

Comment:
For Kubernetes we don't use the `redpanda.yaml` file, but rather a configuration file that we can apply to the Kubernetes cluster. So any name can be used, and it can be stored anywhere (as long as when you try to apply it to your network, you have access to said file).

```suggestion
To demonstrate how to get up and running with TLS in Kubernetes, we’ll create a sample configuration file for a one-node cluster. We'll call it `tls_example.yaml` but feel free to name it however you like. You can save it anywhere your Kubernetes network have reading access to it.
```

@coral-waters wdyt?

Reply:
Word. Looks good.